visualization/dashboard/src/MLIDashboard/Controls/CohortEditor/CohortEditor.tsx

```diff
@@ -309,6 +352,7 @@ export class CohortEditor extends React.PureComponent<ICohortEditorProps, ICohor
 
     private setDefaultStateForKey(key: string): void {
         let filter: IFilter = { column: key } as IFilter;
+        filter.arg = [];
```

Comment:
given that this isn't shared or modified dynamically, it may be cleaner toi just have line 365 be filter.arg = [max]